_test/helper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Helper functions for the findologicxmlexport plugin
+ *
+ * @group plugin_findologicxmlexport
+ * @group plugins
+ */
+
+//======================================================================
+// HELPER FUNCTIONS AND CONSTANTS
+//======================================================================
+
+class helper
+{
+    /**
+     * A page needs to have content. This is the placeholder for this value.
+     */
+    const PAGE_CONTENT_PLACEHOLDER = 'This page is for test purposes.';
+
+    /**
+     * Helper function to save a page.
+     *
+     * @param array $ids An array with page ids (eg. start or wiki:syntax)
+     * @param string $content Optional content, else the default test value will be set
+     */
+    public function savePages($ids, $content = self::PAGE_CONTENT_PLACEHOLDER)
```

Comment:
Since this class does not have any state (ie. no member variables) you can define the methods as static. Then you do not have to instantiate a helper instance and can just call `Helper::savePages()`.